src/cosmos/Swap.fixture.tsx

```diff
@@ -64,6 +64,7 @@ function Fixture() {
   const [defaultOutputAmount] = useValue('defaultOutputAmount', { defaultValue: 0 })
 
   const [hideConnectionUI] = useValue('hideConnectionUI', { defaultValue: false })
+  const [hideL2BridgeBanner] = useValue('hideL2BridgeBanner', { defaultValue: false })
```

Comment:
thoughts on making it hidden by default 🤔 thinking of external integrators who have defined custom wrapper styles and hopefully they wont get thrown off by upgrading

Reply:
hmmmm I'm not sure... that makes sense to me, but then I also feel like integrators might not realize this functionality is there unless it's on by default? I'm not sure who should make that decision wdyt

Reply:
okie we can leave as is for now and change it later if needed